pack.mcmeta

```diff
@@ -1,6 +1,6 @@
 {
   "pack": {
-    "pack_format": 2,
+    "pack_format": 3,
```

Comment:
This change will break `soulshardstow/blockstates/BlockCage` and `enderio/blockstates/blockIngotStorage`